.github/workflows/main.yml

```diff
@@ -144,5 +144,14 @@ jobs:
       - run: rye config --set-bool behavior.use-uv=true
       - run: rye build
 
-      - name: Upload package to PyPI
+      - name: Publish logfire to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
+
+      - name: Build logfire-api
+        run: rye build
+        working-directory: logfire-api/
+
+      - name: Publish logfire-api to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages-dir: logfire-api/
```

Comment:
I've created the publisher... Not sure if this works.

<img width="833" alt="Screenshot 2024-06-21 at 17 55 49" src="https://github.com/pydantic/logfire/assets/7353520/ed5bca7c-37a8-4efa-b80d-627d19f4f6f8">